devito/ops/operator.py

```diff
@@ -26,22 +30,45 @@ def _specialize_iet(self, iet, **kwargs):
         ops_partition = Call(namespace['ops_partition'], Literal('""'))
         ops_exit = Call(namespace['ops_exit'])
 
+        ops_block = OpsBlock('block')
+
         dims = []
+        to_dat = set()
```

Comment:
watch out, this might lead to non-deterministic code gen. You may wanna use
```
from devito.tools import filter_sorted
```
?